distribution/docs/src/main/resources/_contents/_catalog-contents/integrating-catalog-contents.adoc

```diff
@@ -1817,3 +1819,50 @@ Example:
     ]
 }
 ----
+
+==== Attribute Injection Definition
+
+Attribute injections are defined attributes that will be injected into all metacard types or into specific metacard types. Essentially, they allow you to extend existing metacard types with new attributes.
```

Comment:
Nit - this can be simplified for clarity.  Replace "Essentially..." with "This allows metacard types to be extended with new attributes".